content/command-line/concepts/bash/terms/input-output-redirection/input-output-redirection.md

```diff
@@ -0,0 +1,42 @@
+---
+Title: 'Bash: Input-Output Redirection'
+Description: 'Redirection allows the modification of the destination of standard input and standard output in a shell.'
+Subjects:
+  - 'Bash/Shell'
+  - 'Computer Science'
+Tags:
+  - 'Bash/Shell'
+  - 'Command Line'
+  - 'Linux'
+  - 'Redirection'
+CatalogContent:
+  - 'learn-the-command-line'
+  - 'paths/computer-science'
+---
+
+Redirection is a form of input-output manipulation in a shell program such as [Bash](https://www.codecademy.com/resources/docs/command-line/bash). Redirection allows changing the values of `stdout` and `stdin` in a shell to a different location, i.e., a file can be read to `stdin` instead of taking input from keyboard and the output can be written to a file instead of going to terminal which is the default value of `stdout` in most cases.
+To perform redirection special notations interpreted by the shell are used. The `<` and `>` operators are used for output and input redirections respectively.
```

Comment:
This can be better written in list format - 

```suggestion
Special operators are used for redirection:

- `<` redirects input.
- `>` redirects output.
```